sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -185,15 +187,13 @@ def send(self, events, **kwargs):
                 ]
             except AttributeError:
                 pass  # means it's a dictionary
-            kwargs.setdefault(
-                "content_type", "application/cloudevents-batch+json; charset=utf-8"
-            )
+            content_type = "application/cloudevents-batch+json; charset=utf-8"
```

Comment:
do you still want users to be able to set `content_type`?

Reply:
yes - updated :P

Reply:
I think you're overriding the content type you got from users here